protostar/testing/test_runner.py

```diff
@@ -177,12 +181,14 @@ async def _invoke_test_cases(
         execution_state: TestExecutionState,
     ) -> None:
         for test_case in test_suite.test_cases:
-            test_result = await self._invoke_test_case(test_case, execution_state)
+            test_result = await self._invoke_test_case(
+                test_case, execution_state, profiling=self.profiling
+            )
             self.shared_tests_state.put_result(test_result)
 
     @staticmethod
     async def _invoke_test_case(
-        test_case: TestCase, initial_state: TestExecutionState
+        test_case: TestCase, initial_state: TestExecutionState, profiling=False
```

Comment:
I am thinking if it wouldn't make more sense to put this flag inside `TestConfig`. Much less passing via args and it would be possible to enable via cheatcodes in setup.

Reply:
Yess, maybe a good idea would be moving other flags there as well? e.g. `disable_hint_validation_in_user_contracts`